nats.go

```diff
@@ -911,6 +967,33 @@ func ClientCert(certFile, keyFile string) Option {
 	}
 }
 
+// ClientCertMem is a helper option to provide the client certificate from memory.
+// If Secure is not already set this will set it as well.
+func ClientCertMem(certFile []byte, keyFile []byte) Option {
```

Comment:
Invalid param names, those are not files
```suggestion
func ClientCertMem(cert []byte, key []byte) Option {
```